torchvision/models/feature_extraction.py

```diff
@@ -431,17 +433,21 @@ def create_feature_extractor(
         }
     is_training = model.training
 
-    assert any(
-        arg is not None for arg in [return_nodes, train_return_nodes, eval_return_nodes]
-    ), "Either `return_nodes` or `train_return_nodes` and `eval_return_nodes` together, should be specified"
+    if not any(arg is not None for arg in [return_nodes, train_return_nodes, eval_return_nodes]):
 
-    assert not (
-        (train_return_nodes is None) ^ (eval_return_nodes is None)
-    ), "If any of `train_return_nodes` and `eval_return_nodes` are specified, then both should be specified"
+        raise RuntimeError(
+            "Either `return_nodes` or `train_return_nodes` and `eval_return_nodes` together, should be specified"
+        )
 
-    assert (return_nodes is None) ^ (
-        train_return_nodes is None
-    ), "If `train_return_nodes` and `eval_return_nodes` are specified, then both should be specified"
+    if (train_return_nodes is None) ^ (eval_return_nodes is None):
+        raise RuntimeError(
+            "If any of `train_return_nodes` and `eval_return_nodes` are specified, then both should be specified"
+        )
+
+    if not ((return_nodes is None) ^ (train_return_nodes is None)):
+        raise RuntimeError(
+            "If `train_return_nodes` and `eval_return_nodes` are specified, then both should be specified"
```

Comment:
This isn't from this PR, but I'm having a hard time understanding this error message. It reads as

> If A and B are specified, then both should be specified

Also it's metnionning `eval_return_nodes` but it should probably be `return_nodes`